src/components/agencies/agencies.component.js

```diff
@@ -180,15 +203,33 @@ class AgenciesMain extends React.Component {
                           {status}
                         </div>
                       </td>
-                      <td
-                        className="blue-color"
-                        onClick={(e) => {
-                          e.stopPropagation();
-                          goToPage(EDIT_AGENCIES_PAGE, item._id);
```

Comment:
Hi @Azzoo! The edit button was implemented to go to the "Edit Agencies" page (as per #178), but now goes to the "Boardings" page. Were the designs updated?